csp/impl/struct.py

```diff
@@ -105,18 +105,12 @@ def collectts(cls, **kwargs):
 
         return csp.struct_collectts(cls, kwargs)
 
-    @classmethod
-    def _postprocess_dict_to_python(cls, d):
```

Comment:
any reason to leave preprocess_dict around?

Reply:
Nope - removed that too now